Sources/TSCBasic/Process.swift

```diff
@@ -223,6 +226,36 @@ public final class Process: ObjectIdentifierProtocol {
     /// Value: Path to the executable, if found.
     static private var validatedExecutablesMap = [String: AbsolutePath?]()
 
+  #if os(macOS)
+    /// Create a new process instance.
+    ///
+    /// - Parameters:
+    ///   - arguments: The arguments for the subprocess.
+    ///   - environment: The environment to pass to subprocess. By default the current process environment
+    ///     will be inherited.
+    ///   - workingDirectory: The path to the directory under which to run the process.
+    ///   - outputRedirection: How process redirects its output. Default value is .collect.
+    ///   - verbose: If true, launch() will print the arguments of the subprocess before launching it.
+    ///   - startNewProcessGroup: If true, a new progress group is created for the child making it
+    ///     continue running even if the parent is killed or interrupted. Default value is true.
+    @available(macOS 10.15, *)
```

Comment:
Glibc 2.29+ has `posix_spawn_file_actions_addchdir_np` as well (same signature), so this can work on Linux if you want to do that. I'm not sure which distros have which versions (2.29 was released Feb 2019) so it might need an `#if __GLIBC_PREREQ(2, 29)` depending on what Linux flavors the SwiftUI supports.